src/js/components/Video/Video.js

```diff
@@ -413,6 +441,33 @@ const Video = forwardRef(
   },
 );
 
+Video.propTypes = {
```

Comment:
Can we move the propTypes to the doc.js file in order to be consistent with the rest of the components?